src/components/ReplyFormAlert.js

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+import cx from 'classnames';
+
+const ReplyFormAlert = ({
+  type,
+  icon,
+  children
+}) => (
+  <div className="edit-alert">
+    <div
+      className={cx({
+        alert: true,
+        [type]: true
```

Comment:
Not wrong but seems a bit weird to me. If the two possibilities are `error` and `success` then wouldn't it be clearer to have a boolean prop called `success` and put `success: success, error: !success` here?